### ISSUE TITLE
Implement function "DeleteAllReports"

### ISSUE THREAD

**Issue by GiadaVent** [pyansys/pyaedt#1990]
### Description of the feature

Would be nice to have the equivalent function of: 
"oModule.DeleteAllReports()"
for Maxwell 2D and 3D.

**Comment by Samuelopez-ansys** [comment]
Hi @GiadaVent ,

I have implemented a new feature in the method: delete_report, the option to delete all reports if the user does not pass any name. 
For example:
aedtapp.post.delete_report()

I will close this issue once the branch is merged in the main.

Thanks
